MediaPlayer/tests/MediaPlayer-specs.js

```diff
@@ -105,4 +115,272 @@ describe('MediaPlayer', () => {
+			render(
+				<MediaPlayer data-testid="media-player" onPrevious={handlePrevious}>
+					{audioFiles.map((audioFile, index) => (<source key={index} src={audioFile} type="audio/mp3" />))}
+				</MediaPlayer>
+			);
+			const audioSource = screen.getByTestId('media-player').children.item(0).children.item(0);
+			const nextButton = screen.getByLabelText('Next');
+			const previousButton = screen.getByLabelText('Previous');
+
+			userEvent.click(nextButton); // go to next audio media - audioFiles[1]
+			userEvent.click(previousButton); // go back to first audio media - audioFiles[0]
+
+			const expected = 'https://sampleswap.org/mp3/artist/254731/BossPlayer_Your-Right-Here-160.mp3';
+
+			expect(audioSource).toHaveProperty('src', expected);
+		});
+
+		test('should play last audio media in the playlist when `Previous` button is clicked and first media in playlist is playing and `repeat=all`', () => {
+			const handlePrevious = jest.fn();
+			// `onPrevious` added for code coverage purposes
+			render(
+				<MediaPlayer data-testid="media-player" onPrevious={handlePrevious}>
+					{audioFiles.map((audioFile, index) => (<source key={index} src={audioFile} type="audio/mp3" />))}
+				</MediaPlayer>
+			);
+
+			const previousButton = screen.getByLabelText('Previous');
+
+			const repeatButton = screen.getByLabelText('Repeat');
+
+			userEvent.click(repeatButton);
+			userEvent.click(repeatButton);
+
+			userEvent.click(previousButton); // go back to last audio media - audioFiles[5]
+
+			const audioSource = screen.getByTestId('media-player').children.item(0).children.item(0);
+			const expected = 'https://sampleswap.org/mp3/artist/47067/DJ-Masque_Dont-Forget-To-Be-Yourself-160.mp3';
+
+			expect(audioSource).toHaveProperty('src', expected);
+		});
+
+		test('should restart the playlist when `Next` button is clicked and last media in playlist is playing and `repeat=all`', () => {
+			const handleNext = jest.fn();
+			// `onNext` added for code coverage purposes
+			render(
+				<MediaPlayer data-testid="media-player" onNext={handleNext}>
+					{audioFiles.map((audioFile, index) => (<source key={index} src={audioFile} type="audio/mp3" />))}
+				</MediaPlayer>
+			);
+
+			const nextButton = screen.getByLabelText('Next');
+
+			const repeatButton = screen.getByLabelText('Repeat');
+
+			userEvent.click(repeatButton);
+			userEvent.click(repeatButton);
+
+			userEvent.click(nextButton);
+			userEvent.click(nextButton);
+			userEvent.click(nextButton);
+			userEvent.click(nextButton);
+			userEvent.click(nextButton);
+			userEvent.click(nextButton); // go back to first audio media - audioFiles[0]
+
+			const audioSource = screen.getByTestId('media-player').children.item(0).children.item(0);
+			const expected = 'https://sampleswap.org/mp3/artist/254731/BossPlayer_Your-Right-Here-160.mp3';
+
+			expect(audioSource).toHaveProperty('src', expected);
+		});
+
+		test('should not play audio media in order when `Shuffle` button is clicked', () => {
```

Comment:
It would be better to delete this test.
Because the playlist played when 'Shuffle' button is clicked is random, expected value could be second media from audioFiles.

Reply:
Sure, I agree